recipes-extended/spi-minnowmax-board/files/ostro-6lowpan-start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+ip link set wpan0 address ${HWADDR}
+iz set wpan0 ${PAN} ${ADDR} ${CHANNEL}
```

Comment:
Can we check if these were provided and if not then calculate defaults? Like hwaddr could be part of the machine-id. I guess PAN and CHANNEL defaults could be a fixed value so they find other ostro.